docs/lib/auth/escapehatch.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Escape Hatch
+description: Underlying service
+---
+
+Access the escape hatch for Auth category
```

Comment:
This needs to be removed, otherwise, JS will show a section called "Escape Hatch"